core/src/test/java/org/apache/calcite/test/SqlValidatorTest.java

```diff
@@ -7540,6 +7540,47 @@ public void _testGroupExpressionEquivalenceParams() {
         .fails("Column 'C1' not found in any table");
   }
 
+  /**
```

Comment:
We need some negative tests, to prove that UNNEST is illegal in non-Presto conformance.

Consider splitting this method up, and adding comments. It's not clear what each test case is testing.